CIP-0134/README.md

```diff
@@ -0,0 +1,120 @@
+---
+CIP: 134
+Title: Cardano URIs - Address Representation
+Category: Wallets
+Status: Proposed
+Authors:
+  - Steven Johnson <steven.johnson@iohk.io>
+Implementors: []
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/888
+Created: 2024-08-23
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP proposes an extension to [CIP-0013] to allow easy and unambiguous encoding of
+[CIP-0019]/[CIP-0105] Addresses into URL's.
+
+## Motivation: why is this CIP necessary?
+
+[CIP-0013] defines encoding for payment addresses and stake pool, however [CIP-0019]
+and [CIP-0105] define numerous other address types.
+
+These addresses cannot currently be encoded into URIs unambiguously.
+This extension proposes a simple and forward compatible method of encoding such addresses into the URI scheme defined by [CIP-0013].
+
+[x509] certificates can contain name or alternative name information related to either the issuer of
+the certificate or its subject.
+It is desirable to distinguish an Issuer or Subject of a certificate by one or more on-chain keys.
+This, for example, can facilitate the ability to link off-chain actions authorized with a x509 certificate,
+with an on-chain identity.
+
+Currently, there is no standard way to embed a Cardano address, such as a stake address,
+or a dRep address as distinguishing name within a [x509] certificate.
+
+A URI is one of the possible names that can be associated with the Issuer or Subject of a certificate.
+This URI can be referenced in the alternative name for both the Issuer and Subject of a certificate
+by using the uniformResourceIdentifier in the general name.
+[CIP-0013] does not define a method for stake or dRep addresses, etc., to be encoded in the URI scheme it defines.
+
+Allowing these addresses to be easily encoded as URIs allows them to be 100% interoperable
+with existing public key infrastructure and certificate creation tools.
+
+## Specification
+
+We extend [CIP-0013] with a single new authority for referencing Cardano addresses in [CIP-0019] format.
+
+### Grammar & interpretation
+
+We extend the grammar from [CIP-0013] with the new authority:
+
+```
+authorityref = (... | addr )
+
+addr = "//addr/" cip19-addr
+cip19-addr = *cip19-char
+cip19-char = ALPHA / DIGIT / "_"
+```
+
+Effectively, any address string specified by [CIP-0019], [CIP-0105] or future extension to either
+of these specifications can be embedded directly within the URI.
+
+### Examples
+
+#### CIP-19 Addresses
+
+```uri
+web+cardano://addr/addr1qx2fxv2umyhttkxyxp8x0dlpdt3k6cwng5pxj3jhsydzer3n0d3vllmyqwsx5wktcd8cc3sq835lu7drv2xwl2wywfgse35a3x
+web+cardano://addr/addr_test1gz2fxv2umyhttkxyxp8x0dlpdt3k6cwng5pxj3jhsydzer5pnz75xxcrdw5vky
+web+cardano://addr/stake1uyehkck0lajq8gr28t9uxnuvgcqrc6070x3k9r8048z8y5gh6ffgw
+web+cardano://addr/stake_test1uqehkck0lajq8gr28t9uxnuvgcqrc6070x3k9r8048z8y5gssrtvn
+```
+
+#### CIP-105 Addresses
```

Comment:
One note on this is that CIP129 suggests changing theses CIP105 prefixes https://github.com/cardano-foundation/CIPs/pull/857